manifests/1.2.0/opensearch-1.2.0.yml

```diff
@@ -1,11 +1,24 @@
+---
+schema-version: "1.0"
 build:
   name: OpenSearch
   version: 1.2.0
 components:
-- name: OpenSearch
-  ref: 1.x
-  repository: https://github.com/opensearch-project/OpenSearch.git
-  checks:
-    - gradle:publish
-    - gradle:properties:version
-schema-version: '1.0'
+  - name: OpenSearch
+    repository: https://github.com/opensearch-project/OpenSearch.git
+    ref: "1.x"
+    checks:
+      - gradle:publish
+      - gradle:properties:version
+  - name: common-utils
```

Comment:
common-utils and job-scheduler depend on OpenSearch, and should be using maven for 1.2; I would remove them from this PR too, and get that working before re-adding them

Reply:
Got you.

Reply:
But since 1.x is already the old content of 1.2.0 manifest, I can simply do a manual trigger in Jenkins, without touching this PR I think.

Reply:
I dislike running anything manually but what’s done is done :)